pkg/dockerregistry/server/quotarestrictedblobstore.go

```diff
@@ -122,7 +123,51 @@ func (bw *quotaRestrictedBlobWriter) Commit(ctx context.Context, provisional dis
 
 // admitBlobWrite checks whether the blob does not exceed image quota, if set. Returns
 // ErrAccessDenied error if the quota is exceeded.
-func admitBlobWrite(ctx context.Context, repo *repository) error {
+func admitBlobWrite(ctx context.Context, repo *repository, size int64) error {
+	if size < 1 {
+		return nil
+	}
+
+	err := admitLimitRanges(ctx, repo, size)
```

Comment:
Whenever possible use short if form, doing so ensures there's no nasty variable shadowing etc. I realize this is not the problem with this method which is short. I'm just saying for future ;) The same applies to the other if, couple lines below.